From 4557b3f6c4273cd83b701beaf7a150c806fed298 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Thu, 19 Feb 2009 00:43:23 +0000
Subject: X11/fonts/fontmod.h doesn't exist anymore.

libXfont 1.4.0 no longer includes this file, nor do we need it, so just stop
attempting to reference it.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
diff --git a/hw/xfree86/loader/loaderProcs.h b/hw/xfree86/loader/loaderProcs.h
index 827f3a9..a10f0b3 100644
--- a/hw/xfree86/loader/loaderProcs.h
+++ b/hw/xfree86/loader/loaderProcs.h
@@ -56,7 +56,6 @@
 #undef IN_LOADER
 #define IN_LOADER
 #include "xf86Module.h"
-#include <X11/fonts/fontmod.h>
 
 typedef struct module_desc {
     struct module_desc *child;
--
cgit v0.8.2
